cartoframes/utils/metrics.py

```diff
@@ -100,22 +111,32 @@ def wrapper_func(*args, **kwargs):
             result = func(*args, **kwargs)
 
             if get_metrics_enabled():
-                extra_metrics_data = build_extra_metrics_data(func, *args, **kwargs)
-                post_metrics(event_name, extra_metrics_data)
+                extra_metrics_data, server_domain_tld = build_extra_metrics_data(func, *args, **kwargs)
+                post_metrics(event_name, extra_metrics_data, server_domain_tld)
 
             return result
         return wrapper_func
     return decorator_func
 
 
 def build_extra_metrics_data(decorated_function, *args, **kwargs):
+    extra_metrics = {}
+    server_domain_tld = PROD_DOMAIN_TLD
+
     try:
-        credentials = get_parameter_from_decorator(
-            'credentials', decorated_function, *args, **kwargs)
+        credentials = get_parameter_from_decorator('credentials', decorated_function, *args, **kwargs)
         credentials = get_credentials(credentials)
-        return {'user_id': credentials.user_id} if credentials and credentials.user_id else {}
+
+        hostname = urlparse(credentials.base_url).hostname
+        server_domain_tld = '.'.join(hostname.split('.')[-2:])
```

Comment:
Move this to a function `utils.get_server_domain_tld(base_url)` or `get_metrics_server(base_url)` (this file)

Reply:
:rocket: